code/config.yaml

```diff
@@ -0,0 +1,28 @@
+llm_model:
```

Comment:
Is there a reason you added this file? Typically, this will get automatically generated from the sample_config.yaml file. I think the .gitignore is supposed to keep this file excluded as it will be different from person-to-person.

Reply:
ahh. I didn't realize it was automatically generated by the sample_config.yaml

I added it because when deploying the NIMs via compose, the URL for the local LLM NIM needs to change from "http://llm-nim-0:8000/v1" to "http://llm-nim:8000/v1" 

I can remove it and make the changes in sample_config.yaml